dashboard/config/levels/custom/dance/dance_ai_try_at_measure.level

```diff
@@ -105,7 +105,7 @@
           <field name="UNIT">"measures"</field>
           <next>
             <block type="Dancelab_ai" id="AZFax0W2U%3B*@TX0l4o">
-              <field name="VALUE">{"inputs":["spooky","black-and-white","robot"],"backgroundEffect":"squiggles","backgroundColor":"rave","foregroundEffect":"bubbles"}</field>
+              <field name="VALUE">{"inputs":["ghost","chequered-flag","robot"],"backgroundEffect":"squiggles","backgroundColor":"rave","foregroundEffect":"bubbles"}</field>
```

Comment:
NIT: Spelling error? Was it supposed to be checkered-flag?

Reply:
Spelling is standard for the emoji - https://emojipedia.org/chequered-flag